code/style.css

```diff
@@ -1,4 +1,208 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+html{
+    background: #ffffff;
+    color: #000000;
+    font-family: sans-serif;
+  }
+  .container{
+    display: none;
+  }
+  header {
+    display: flex;
+    flex-direction: column;
+  }
+  header img {
+    align-self: center;
+    width: 40%;
+  }
+  .logo{
+    display: none;
+  }
+  h1 {
+    display: none;
+    margin-bottom: 15px;
+  }
+  h2 {
+    text-align: center;
+  } 
+  h3{
+    margin-bottom: 2px;
+  }
+  .nav-links {
+    display: none;
+  }
+  .nav-links li {
+    list-style: none;
+  }
+  p {
+    padding: 5px;
+  }
+  a {
+    text-decoration: none;
+    font-weight: bold;
+    align-content: space-around;
+    padding: 10px;
+    color: #000000;
+  }
+  a:hover{
+    color: #888888;
+  }
+  
+  .hamburger{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 80px;
+    height: 80px;
+    transition: all .5s ease-in-out;
+  }
+  .line{
+    width: 50px;
+    height: 6px;
+    background: #000;
+    border-radius: 5px;
+    transition: all .5s ease-in-out;
+  }
+  .line::before, .line::after{
+    content: '';
+    position: absolute;
+    width: 50px;
+    height: 8px;
+    background: #000;
+    border-radius: 5px;
+    transition: all .5s ease-in-out;
+  }
+  .line::before{
+    transform: translateY(-16px);
+  }
+  .line::after{
+    transform: translateY(16px);
+  }
+  .hamburger.open .line{
+    transform: translateX(-50px);
+    background: transparent;
+  }
+  .hamburger.open .line::before{
+    transform: rotate(45deg) translate(35px, -35px);
+  }
+  .hamburger.open .line::after{
+    transform: rotate(-45deg) translate(35px, 35px);
+  }
+  .big-news{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+  }
+  .big-news img{
+    width: 100%;
+  }
```

Comment:
Would look extra good if the `big-news img` were aligned with the `card-container`!